plugins/modules/ndo_ipsla_monitoring_policy.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: ndo_ipsla_monitoring_policy
+short_description: Manage IPSLA Monitoring Policies on Cisco Nexus Dashboard Orchestrator (NDO).
```

Comment:
Shall we write IP SLA with a space in between since this is how it is typically documented in other docs?

```suggestion
short_description: Manage IP SLA Monitoring Policies on Cisco Nexus Dashboard Orchestrator (NDO).
```